Cargo.toml

```
@@ -76,7 +76,7 @@ web-time = { workspace = true, optional =  true }
 default = ["sys"]
 log = ["dep:log"]
 compiled_data = ["tzdb"]
-sys = ["std", "dep:web-time"]
+sys = ["std", "dep:web-time", "iana-time-zone"]
```

```suggestion
sys = ["std", "dep:web-time", "dep:iana-time-zone"]
```

I think this needs a dep prefix. It's treating `iana-time-zone` as a feature in CI.